### ISSUE TITLE
Compilation fail unless installing erlang-eunit

### ISSUE THREAD

**Issue by Nagasaki45** [bitwalker/alpine-elixir-phoenix#5]
I'm new to elixir/phoenix and not familiar with erlang at all.
I have [`eredis`](https://hex.pm/packages/eredis) as a dependency in my project, and when I try to compile it I get:

```
===> Compiling idna
==> eredis (compile)
src/eredis_parser.erl:none: undefined parse transform 'eunit_autoexport'
Compiling src/eredis_parser.erl failed:
ERROR: compile failed while processing /opt/app/deps/eredis: rebar_abort
** (Mix) Could not compile dependency :eredis, "/opt/app/.mix/rebar compile skip_deps=true deps_dir="/opt/app/_build/prod/lib"" command failed. You can recompile this dependency with "mix deps.compile eredis", update it with "mix deps.update eredis" or clean it with "mix deps.clean eredis"
The command '/bin/sh -c mix compile' returned a non-zero code: 1
```

I installed `erlang-eunit` and it solved the problem.

```
RUN apk --no-cache add erlang-eunit
```

**Comment by bitwalker** [comment]
I need to update this image to use my alpine-elixir image as the base, which will solve this problem, the current image does not contain all erlang applications in an attempt to be lighter weight, leaving the installation of those optional packages up the consumer, but my elixir image builds erlang from source, and includes everything, so it takes a different approach.

**Comment by Nagasaki45** [comment]
As far as I can tell the issue is solved by c18fb016b14dfc9916cf9f22606e5593ec3cbfb0. Isn't it?

**Comment by bitwalker** [comment]
Yes, forgot to update this issue, sorry about that!
